site/content/en/docs/Installation/terraform.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Install Agones using Terraform"
+linkTitle: "Install with Terraform"
+weight: 4
+description: >
+  This chart install the Agones application and defines deployment on a [Kubernetes](http://kubernetes.io) cluster using the Terraform.
+
+---
+
+## Prerequisites
+
+- Terraform v0.11.13
+- [Helm](https://docs.helm.sh/helm/) package manager 2.10.0+
+- Access to Google Cloud Kubernetes Engine
+- `gcloud` utility installed
+- Git
+
+## Installing the Agones as Terraform submodule
+
+You can use Terraform to provision your GKE cluster and install agones on it using Helm Terraform provider.
```

Comment:
Can we add a quick summary of what this does? I.e. creates  a cluster with n nodes, sets up rbac rules, install Agones, etc.

Also, do we need a list of configurable params?

Reply:
Yes, I think it is neccessary to add those params.